docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -112,61 +114,107 @@ After the active instance has been configured, you can configure one or more sta
+
+- Click **Make Active** on the standby. You should see a list of available backups that you can restore from. (Don't promote the standby.)
+- Verify that Prometheus on the standby is able to see similar metrics to the active. Navigate to `http://<STANDBY_IP>:9090/targets`; the federate target should have a status of UP, and the endpoint should match the active instance IP address.
+
+    ![Verify Prometheus](/images/yp/high-availability/ha-prometheus.png)
+
+- Verify that the standby has all the database releases that are in use by universes also listed as Active on the **Releases** page (navigate to **Profile > Releases**). To discover all the database releases that are in use by universes, you can view the **Dashboard** page.
+
+    If releases are missing, follow the instructions in [How to configure YugabyteDB Anywhere to provide Older, Hotfix, or Debug Builds](https://support.yugabyte.com/hc/en-us/articles/360054421952-How-to-configure-YugabyteDB-Anywhere-to-provide-Older-Hotfix-or-Debug-Builds).
+
+If the YBA instances are configured to use the HTTPS protocol and you are having problems, verify that certificates and ports are set up correctly. If the issue persists, consider relaxing the certificate validation requirements as a workaround, by enabling the [runtime configuration](../manage-runtime-config/) `yb.ha.ws.ssl.loose.acceptAnyCertificate` (set the flag to `true`).
+
 ## Promote a standby instance to active
 
 You can make a standby instance active as follows:
 
-1. Open **Replication Configuration** of the standby instance that you wish to promote to active and click **Make Active**.
+1. On the standby instance you want to promote, navigate to **Admin > High Availability > Replication Configuration** and click **Make Active**.
 
-1. Use the **Make Active** dialog to select the backup from which you want to restore (in most cases, you should choose the most recent backup) and enable **Confirm promotion**.
+1. Select the backup from which you want to restore (in most cases, you should choose the most recent backup) and enable **Confirm promotion**.
 
-1. Click **Continue**. The restore takes a few seconds, after which expect to be logged out.
+1. Click **Continue**. The restore takes a few seconds, after which expect to be signed out.
 
-1. Log in using credentials that you had configured on the previously active instance.
+1. Sign in using the credentials that you had configured on the previously active instance.
 
 You should be able to see that all of the data has been restored into the instance, including universes, users, metrics, alerts, task history, cloud providers, and so on.
 
-## Check results
+### Verify promotion
+
+After switching or failing over to the standby, verify that the old active YBA instance is in standby mode (switchover) or no longer available (failover). If both YBA instances were to attempt to perform actions on a universe, it could have unpredictable side effects.
```

Comment:
```suggestion
After switching or failing over to the standby, verify that the old active YBA instance is in standby mode (switchover), or no longer available (failover). If both YBA instances were to attempt to perform actions on a universe, it could have unpredictable side effects.
```